pkg/apis/networking/validation/validation.go

```diff
@@ -68,11 +68,21 @@ func ValidateNetworkPolicyPort(port *networking.NetworkPolicyPort, portPath *fie
 			for _, msg := range validation.IsValidPortNum(int(port.Port.IntVal)) {
 				allErrs = append(allErrs, field.Invalid(portPath.Child("port"), port.Port.IntVal, msg))
 			}
+			if *port.EndPort < port.Port.IntVal {
+				allErrs = append(allErrs, field.Invalid(portPath.Child("endPort"), port.Port.IntVal, "endPort field must be equal or bigger than port field"))
+			}
 		} else {
+			if port.EndPort != nil {
```

Comment:
Can we add unit tests for validating endPort?

Reply:
Added a bunch of them, can add more if necessary